openqa-maintenance.py

```diff
@@ -158,12 +158,37 @@ def kgraft_target(req):
 
         return target, action
 
+    @staticmethod
+    def parse_kgraft_version(kgraft_target):
+        return kgraft_target.lstrip('SLE').split('_')[0]
+
+    @staticmethod
+    def kernel_target(req):
+        if req:
+            for a in req.actions:
+                # kernel incidents have kernel-source package (suprise!)
+                if a.src_package.startswith('kernel-source'):
+                    return True, a
+        return None, None
+
     def settings(self, src_prj, dst_prj, packages, req=None):
         settings = super(SUSEUpdate, self).settings(src_prj, dst_prj, packages, req)
 
-        # special handling for kgraft incidents
+        # special handling for kgraft and kernel incidents
         if settings['FLAVOR'] in ('KGraft', 'Server-DVD-Incidents-Kernel'):
             kgraft_target, action = self.kgraft_target(req)
+		# Server-DVD-Incidents-Incidents handling
+        if settings['FLAVOR'] == 'Server-DVD-Incidents-Kernel':
+            kernel_target, kaction = self.kernel_target(req)
+            if kernel_target or kgraft_target:
+                incident_id = incident_id = re.match(r".*:(\d+)$", action.src_project).group(1) if kgraft_target else re.match(r".*:(\d+)$", kaction.src_project).group(1)
```

Comment:
I would prefer if you split this statement in a way it's readable in github without scrolling